opennms-webapp/src/main/webapp/includes/bootstrap-footer.jsp

```diff
@@ -80,6 +81,45 @@
                 %>
             </p>
         </footer>
+
+        <c:if test="${!empty pageContext.request.remoteUser}">
```

Comment:
I'm wondering if there's a better place to put this logic - seems out of place to include this Javascript in the JSP of the footer.

Can we find a way to move this to a .js file instead?